connectors/kafka/element-templates/kafka-inbound-connector.json

```diff
@@ -0,0 +1,226 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "Kafka Consumer Connector",
+  "id": "io.camunda.connectors.inbound.KAFKA.v1",
```

Comment:
Shall we call it maybe more explicit, instead `inbound` => `polling`?

Reply:
I am not sure about this. In the rabbitmq we use inbound. I would like to be consistent. WDYT?

Reply:
Lets go with inbound.

Reply:
Sounds good to me.